integrationtest/mapper/pojo-standalone-realbackend/src/test/java/org/hibernate/search/integrationtest/mapper/pojo/standalone/realbackend/schema/management/ElasticsearchSchemaManagerExporterIT.java

```diff
@@ -0,0 +1,207 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.integrationtest.mapper.pojo.standalone.realbackend.schema.management;
+
+import static org.hibernate.search.util.impl.test.JsonHelper.assertJsonEquals;
+
+import java.io.IOException;
+import java.lang.invoke.MethodHandles;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import org.hibernate.search.integrationtest.mapper.pojo.standalone.realbackend.testsupport.BackendConfigurations;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.DocumentId;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.FullTextField;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.Indexed;
+import org.hibernate.search.mapper.pojo.standalone.mapping.SearchMapping;
+import org.hibernate.search.mapper.pojo.standalone.session.SearchSession;
+import org.hibernate.search.util.impl.integrationtest.mapper.pojo.standalone.StandalonePojoMappingSetupHelper;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+public class ElasticsearchSchemaManagerExporterIT {
+
+	@Rule
+	public final TemporaryFolder temporaryFolder = new TemporaryFolder();
+	@Rule
+	public StandalonePojoMappingSetupHelper setupHelper =
+			StandalonePojoMappingSetupHelper.withSingleBackend(
+					MethodHandles.lookup(), BackendConfigurations.simple() );
+
+	@Test
+	public void elasticsearch() throws IOException {
+		SearchMapping mapping = setupHelper.start()
+				.withProperty( "hibernate.search.backend.type", "elasticsearch" )
+				.withProperty( "hibernate.search.backend.version_check.enabled", false )
+				.withProperty( "hibernate.search.backend.version", "8.6" )
+
+				.withProperty( "hibernate.search.backends." + Article.BACKEND_NAME + ".type", "elasticsearch" )
+				.withProperty( "hibernate.search.backends." + Article.BACKEND_NAME + ".version_check.enabled", false )
+				.withProperty( "hibernate.search.backends." + Article.BACKEND_NAME + ".version", "8.6" )
+				.setup( Book.class, Article.class );
+
+		Path directory = temporaryFolder.newFolder().toPath();
+		try ( SearchSession session = mapping.createSession() ) {
+			session.schemaManager().exportSchema( directory );
+		}
+
+		String bookIndex = Files.readString(
+				directory.resolve( "backend" ) // as we are using the default backend
+						.resolve( "indexes" )
+						.resolve( Book.class.getName() ) // we use FQN as who knows maybe someone will decide to have same class names in different packages
```

Comment:
Technically this should be the index name, which defaults to the entity name, which defaults to the simple name of the class, and can be configured this in the mapper. See: https://github.com/hibernate/hibernate-search/blob/37121f38cfb368d5654534c7e28ee46656895b75/integrationtest/mapper/pojo-base/src/test/java/org/hibernate/search/integrationtest/mapper/pojo/mapping/definition/IndexedBaseIT.java#L67-L126

We shouldn't be using the FQCN here, unless that's what the user selected explicitly.